src/components/core/text/MqText.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { ReactElement } from 'react'
+
+import { Link } from 'react-router-dom'
+import {THEME_EXTRA} from '../../../helpers/theme'
+import { Theme } from '@material-ui/core/styles/createMuiTheme'
+import { WithStyles, createStyles, fade } from '@material-ui/core'
+import classNames from 'classnames'
+import withStyles from '@material-ui/core/styles/withStyles'
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      lineHeight: 1.5,
+      fontSize: '.875rem',
+      fontFamily: '\'Graphik\', sans-serif',
```

Comment:
Can we use this font here?

Reply:
Thanks for pointing this out.
We are now back to `Source Code Pro` and `Karla` which are both free google fonts.